cmd/appsctl/main.go

```diff
@@ -28,23 +31,43 @@ var rootCmd = &cobra.Command{
 	Use:   "appsctl",
 	Short: "A tool to manage Mattermost apps.",
 	PersistentPreRun: func(cmd *cobra.Command, args []string) {
+		log.SetLevel(logrus.InfoLevel)
 		if verbose {
 			log.SetLevel(logrus.DebugLevel)
 		}
+		if quiet {
+			log.SetLevel(logrus.ErrorLevel)
+		}
 	},
 }
 
-func createAWSClient() (aws.Client, error) {
-	accessKey := os.Getenv("APPS_PROVISION_AWS_ACCESS_KEY")
-	secretKey := os.Getenv("APPS_PROVISION_AWS_SECRET_KEY")
+func AsProvisioner() (upaws.Client, error) {
```

Comment:
What is the difference to `AsTest`?

Reply:
Fixed, was a typo, and a very unfortunate one. Thx @hanzei for catching!